tests/utilities.ts

```diff
@@ -1,22 +1,22 @@
-import { MacaroonsBuilder } from 'macaroons.js'
 import { randomBytes } from 'crypto'
 
 import { invoice } from './data'
 import { Identifier } from '../src'
 import { getIdFromRequest } from '../src/helpers'
+import * as Macaroon from "macaroon";
 
-export class BuilderInterface extends MacaroonsBuilder {}
-export function getTestBuilder(secret: string): BuilderInterface {
+export function getTestBuilder(secret: string) {
```

Comment:
Does the typescript not require a return type?

Reply:
Type inference is used when explicit types aren't in place. I also had some issues trying to explicitly list the new macaroon class as a return type, so when I found out that this worked well, I left out the return type.